src/attributes.jl

```diff
@@ -178,8 +184,7 @@ function h5readattr(filename, name::AbstractString)
     file = h5open(filename,"r")
     try
         obj = file[name]
-        a = attributes(obj)
-        dat = Dict(x => read(a[x]) for x in keys(a))
+        dat = Dict(attrs(obj))
```

Comment:
```suggestion
        dat = Dict(AttributeDict(obj))
```

Internally, we should just use the constructor.

Reply:
Is there any difference?

Reply:
Currently,  there is one less level of indirection. For no difference we need `const attrs = AttributeDict`.